pages/print_labels/destination_plates.vue

```diff
@@ -71,43 +60,27 @@ export default {
   },
   data() {
     return {
-      status: statuses.Idle,
-      alertMessage: '',
       printer: this.printers[0],
       numberOfBarcodes: '1',
     }
   },
   computed: {
-    // TODO: GPL-828 - abstract and create functions dynamically.
-    isIdle() {
-      return this.status === statuses.Idle
-    },
-    isSuccess() {
-      return this.status === statuses.Success
-    },
-    isError() {
-      return this.status === statuses.Error
-    },
     isBusy() {
-      return this.status === statuses.Busy
+      return this.$refs.statusAlert?.isBusy || false
     },
   },
   methods: {
-    setStatus(status, message) {
-      this.status = statuses[status]
-      this.alertMessage = message
-    },
     async printLabels() {
-      this.setStatus('Busy', 'Printing labels ...')
+      this.$refs.statusAlert.setStatus('Busy', 'Printing labels ...')
```

Comment:
Oh it sets it here! 👍  Maybe this line is missing from a few of the other printing pages?